packages/land/test/common/ERC721.behavior.ts

```diff
@@ -0,0 +1,758 @@
+          await loadFixture(setupLand);
+        await expect(
+          LandAsOwner.safeBatchTransferFrom(
+            landOwner,
+            TestERC721TokenReceiver,
+            tokenIds,
+            '0x',
+          ),
+        ).to.not.be.reverted;
+        expect(
+          await LandAsOwner.balanceOf(TestERC721TokenReceiver),
+        ).to.be.equal(tokenIds.length);
+      });
+    });
+
+    describe('transfer', function () {
+      it('transferring one NFT results in one erc721 transfer event', async function () {
+        const {LandAsOwner, landOwner, other, tokenIds} =
+          await loadFixture(setupLand);
+        await expect(LandAsOwner.transferFrom(landOwner, other, tokenIds[0]))
+          .to.emit(LandAsOwner, 'Transfer')
+          .withArgs(landOwner, other, tokenIds[0]);
+      });
+      it('transferring one NFT change to correct owner', async function () {
+        const {LandAsOwner, landOwner, other, tokenIds} =
+          await loadFixture(setupLand);
+        await LandAsOwner.transferFrom(landOwner, other, tokenIds[0]);
+        expect(await LandAsOwner.ownerOf(tokenIds[0])).to.be.equal(other);
+      });
+
+      it('transferring one NFT increase new owner balance', async function () {
+        const {LandAsOwner, landOwner, other, tokenIds} =
+          await loadFixture(setupLand);
+        const balanceBefore = await LandAsOwner.balanceOf(other);
+        await LandAsOwner.transferFrom(landOwner, other, tokenIds[0]);
+        const balanceAfter = await LandAsOwner.balanceOf(other);
+        expect(balanceBefore + 1n).to.be.equal(balanceAfter);
+      });
+
+      it('transferring one NFT decrease past owner balance', async function () {
+        const {LandAsOwner, landOwner, other, tokenIds} =
+          await loadFixture(setupLand);
+        const balanceBefore = await LandAsOwner.balanceOf(landOwner);
+        await LandAsOwner.transferFrom(landOwner, other, tokenIds[0]);
+        const balanceAfter = await LandAsOwner.balanceOf(landOwner);
+        expect(balanceBefore - 1n).to.be.equal(balanceAfter);
+      });
+
+      it('transferring from without approval should fails', async function () {
+        const {LandAsOther, landOwner, other, tokenIds} =
+          await loadFixture(setupLand);
+        await expect(
+          LandAsOther.transferFrom(landOwner, other, tokenIds[0]),
+        ).to.be.revertedWith(errorMessages.UNAUTHORIZED_TRANSFER);
+      });
+
+      it('transferring to zero address should fails', async function () {
+        const {LandAsOwner, landOwner, tokenIds} = await loadFixture(setupLand);
+        await expect(
+          LandAsOwner.transferFrom(landOwner, ZeroAddress, tokenIds[0]),
+        ).to.be.revertedWith(errorMessages.NOT_TO_ZEROADDRESS);
+      });
+
+      // TODO: This is right??? do not accept???
+      it('transferring to a contract that do not accept erc721 token should not fail', async function () {
```

Comment:
this has to be a mistake

`transferring to a contract that accepts erc721 token should not fail`